### ISSUE TITLE
Enable behaviour for Ubuntu's Unity notifications

### ISSUE THREAD

**Issue by matterdoc** [mattermost/docs#3237]
Mattermost user `guillermo.vaya` from https://community-release.mattermost.com has requested the following be documented:

```
ok, figured out why. if you have notifiactions with its content enabled, seems to work properly, but if you don't show the content, nothing happens when you click on it
```

See the original post [here](https://community-release.mattermost.com/_redirect/pl/oopx7ufdtfftfrkhfbgbq6dowy).

_This issue was generated from [Mattermost](https://mattermost.com) using the [Doc Up](https://github.com/jwilander/mattermost-plugin-docup) plugin._

**Comment by AnshMishra2001** [comment]
Can I work on this issue ?

**Comment by cwarnermm** [comment]
@Willyfrog - I'm having trouble accessing the original post containing additional context for this issue. Can you summarize the issue please (assuming it continues to be active/present/something we want to communicate in product documentation)?

**Comment by Willyfrog** [comment]
@cwarnermm it was linking to community-release which is no longer available, renaming the link to comunity or community-daily, works: https://community-daily.mattermost.com/core/pl/638bqx9pspgwugmgyz4kpe6ike

**Comment by cwarnermm** [comment]
Thanks, @Willyfrog! Looks like notifications without content weren't clickable on Unity. Do you know if that continues to be the case?
